src/drivers/uavcan/rgbled.cpp

```diff
@@ -43,6 +44,18 @@ UavcanRGBController::UavcanRGBController(uavcan::INode &node) :
 
 int UavcanRGBController::init()
 {
+	// Get parameters
+	int32_t i = 0;
+	param_get(param_find("UAVCAN_LGT_ANTCL"), &i);
```

Comment:
This would be a little safer with ModuleParams like this. 
https://github.com/PX4/Firmware/blob/945c17bc3f875a365f9ea9c9afada6333edf7815/src/modules/mc_rate_control/MulticopterRateControl.hpp#L63
https://github.com/PX4/Firmware/blob/945c17bc3f875a365f9ea9c9afada6333edf7815/src/modules/mc_rate_control/MulticopterRateControl.hpp#L135-L172


It ensures the parameter actually exists in the system (no typos) and the type is correct.

Reply:
Thanks for the tip, that is much cleaner. See changes as of the latest commit.